### DISCUSSION TITLE
写了一个基于Win10 hyper v 虚拟机的PaoPaoDNSGateway 速建说明，基于Alpine linux，喜欢的可以试试

### DISCUSSION THREAD

**Discussion by gukey** [kkkgo/PaoPaoGateWay#7] [配置分享]
**Win 10 hyper-v 虚拟机 PaoPaoDNSGateway 速建说明：**
[123盘](https://www.123pan.com/s/UtJ9-dydtv.html )
**百度网盘：** https://pan.baidu.com/s/1nWJZpYjIWr59W8v-qvMVAQ 提取码: gqpf
[google网盘](https://drive.google.com/drive/folders/1xYGPdDpdH9Wb_SPAd3bb2QmMA-EQFQHf?usp=sharing)

![](http://forumupload.ru/uploads/001b/b0/1b/8/751054.png)

恩山论坛地址：https://www.right.com.cn/forum/forum.php?mod=viewthread&tid=8293054&page=1#pid18702816

**Comment by kkkgo** [comment]
谢谢分享👍

**Comment by kkkgo** [comment]
50MB的体积应该是加了嗅探的吧，如果你用嗅探的话可以重新拉下镜像再生成，之前改进了嗅探的配置

**Reply by gukey** [reply]
> 50MB的体积应该是加了嗅探的吧，如果你用嗅探的话可以重新拉下镜像再生成，之前改进了嗅探的配置

重新拉取，生成在47MB多

**Reply by kkkgo** [reply]
如有docker pull失败的话，可以用上海交大的镜像。https://mirror.sjtu.edu.cn/docs/docker-registry

**Comment by kbrx93** [comment]
dns 和 网关都非常好用，感谢作者，也感谢 @gukey 教程 🙌

**Comment by Leno709394** [comment]
我就是看了楼主的贴子来到这里的。泡泡是好东西。知道的人太少

**Reply by kkkgo** [reply]
谢谢，如果你有好的使用经验或者心得，也欢迎一起分享👏
